cells/_automation/tasks.nix

```diff
@@ -13,6 +13,7 @@
       github.ci = {
         enable = config.actionRun.facts != {};
         repository = "input-output-hk/cardano-prelude";
+        remote = config.preset.github.lib.readRepository inputs.cells._cloud.library.actionCiInputName null;
```

Comment:
This is the remote to clone from. We need to read it from the input fact in order to support PRs from forks.